src/Pages/Error/NotFound.module.scss

```diff
@@ -0,0 +1,91 @@
+.notfnd {
+  background: url(../../../public/images/404bg.jpeg);
```

Comment:
Told you that we don't use stuffs from public directory in relative path??

Change it to

```css
background-image: url("/images/404.jpeg");
```

Reply:
For reference [read this](https://vitejs.dev/guide/assets.html#the-public-directory)